verticapy/_config/config.py

```diff
@@ -148,6 +153,205 @@ def set_option(key: str, value: Any = None) -> None:
+        html_file = open("figures/_config_config_set_option_4.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_4.html
+
+    Sets the maximum number of rows displayed:
+
+    .. code-block:: python
+
+        set_option("max_rows", 5)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("max_rows", 5)
+        html_file = open("figures/_config_config_set_option_5.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_5.html
+
+    Sets the display to light mode:
+
+    .. code-block:: python
+
+        set_option("mode", "light")
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("mode", "light")
+        html_file = open("figures/_config_config_set_option_6.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_6.html
+
+    Sets the display to full mode:
+
+    .. code-block:: python
+
+        set_option("mode", "full")
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("mode", "full")
+        html_file = open("figures/_config_config_set_option_7.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_7.html
+
+    Turn on the loading bar:
+
+    .. code-block:: python
+
+        set_option("percent_bar", True)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("percent_bar", True)
+        html_file = open("figures/_config_config_set_option_8.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_8.html
+
+    Displays the queries and their execution times:
+
+    .. ipython:: python
+
+        set_option("sql_on", True)
+        set_option("time_on", True)
+        @savefig _config_config_set_option_corr.png
+        titanic.corr()
+
+    Hides the queries and execution times:
+
+    .. ipython:: python
+
+        set_option("sql_on", False)
+        set_option("time_on", False)
+
+    Sets the seed for the random number generator and seeds the random state:
+
+    .. ipython:: python
+
+        set_option("random_state", 2)
+        titanic.sample(0.1).shape()
+
+    Change the graphic colors:
+
+    .. ipython:: python
+
+        set_option("colors", ["blue", "red"])
+
+        @savefig _config_config_set_option_hist.png
+        titanic.hist(["pclass", "survived"])
+
+    Change the temporary schema:
+
+    .. ipython:: python
+
+        set_option("temp_schema", "public")
```

Comment:
.. important:: The temporary schema is utilized to create elements that should be dropped at the end of the function in question. In case of an error, the element might still exist and will need to be manually dropped.